### PR TITLE
Write to disk from clients

### PR DESCRIPTION
**Pull request by nikovacs**
Updated docker-compose.yml to mount shared volume between container and host for writing data.

### CODE REVIEW

**Comment by nikovacs** [comment]
Requires modification to server tests and introduction of new client tests.

An existing, tested method is calling the method to write to disk... This is causing tests to fail it is not permitted to write to `/data`, which would be root on one's personal machine.

**Comment by AlexF301** [comment]
The mock method tests the process of the clients writing to disk. All tests scenarios now pass as expected

File: mirrulations-client/src/mirrclient/client.py [hunk 1]
```diff
@@ -190,6 +191,93 @@ def send_job(self, job, job_result):
...
                      timeout=10)
+        self._handle_results(data)
+
+        # For now, still need to send original put request for Mongo
+        # requests.put(
+        #     f'{self.url}/put_results',
+        #     json=dumps(data['job_id']),
+        #     params={'client_id': self.client_id},
+        #     timeout=10
+        # )
```
Comment (evan-toyberg) [inline mirrulations-client/src/mirrclient/client.py:202]: Commented out for future?

**Review by evan-toyberg — COMMENTED** [review]
Possible changes. Mostly removing commented out code. Also shouldn't need to mount a volume in the work_server because it no longer needs to read/write to disk.

File: mirrulations-client/src/mirrclient/client.py (hunk 1 above)
Comment (nikovacs) [reply mirrulations-client/src/mirrclient/client.py:202]: Yes

**Pull request merged**